keras_cv/layers/preprocessing/rand_augment.py

```diff
@@ -0,0 +1,277 @@
+        augmented_sample["images"] = preprocessing_utils.transform_value_range(
+            augmented_sample["images"], (0, 255), self.value_range
+        )
+        return augmented_sample
+
+
+def auto_contrast_policy(magnitude, magnitude_std):
+    return {}
+
+
+def equalize_policy(magnitude, magnitude_std):
+    return {}
+
+
+def solarize_policy(magnitude, magnitude_std):
+    threshold_factor = core.NormalFactorSampler(
+        mean=(magnitude / 10) * 256,
+        standard_deviation=(magnitude_std / 10.0) * 256,
+        min_value=0,
+        max_value=255,
+    )
+    return {"threshold_factor": threshold_factor}
+
+
+def solarize_add_policy(magnitude, magnitude_std):
+    addition_factor = core.NormalFactorSampler(
+        mean=(magnitude / 10) * 110,
+        standard_deviation=(magnitude_std / 10.0) * 110,
+        min_value=0,
+        max_value=110,
+    )
+    return {"addition_factor": addition_factor, "threshold_factor": 128}
+
+
+def invert_policy(magnitude, magnitude_std):
+    return {"addition_factor": 0, "threshold_factor": 0}
+
+
+def color_policy(magnitude, magnitude_std):
+    factor = core.NormalFactorSampler(
+        mean=magnitude / 10.0,
+        standard_deviation=magnitude_std / 10.0,
+        min_value=0,
+        max_value=1,
+    )
+    return {"factor": factor}
+
+
+def contrast_policy(magnitude, magnitude_std):
+    # TODO(lukewood): should we integrate RandomContrast with `factor`?
+    # RandomContrast layer errors when factor=0
+    factor = max(magnitude / 10, 0.001)
+    return {"factor": factor}
+
+
+def brightness_policy(magnitude, magnitude_std):
+    # TODO(lukewood): should we integrate RandomBrightness with `factor`?
+    return {"factor": magnitude / 10.0}
+
+
+def shear_x_policy(magnitude, magnitude_std):
+    factor = core.NormalFactorSampler(
+        mean=magnitude / 10.0,
+        standard_deviation=magnitude_std / 10.0,
+        min_value=0,
+        max_value=1,
+    )
+    return {"x_factor": factor, "y_factor": 0}
+
+
+def shear_y_policy(magnitude, magnitude_std):
+    factor = core.NormalFactorSampler(
+        mean=magnitude / 10.0,
+        standard_deviation=magnitude_std / 10.0,
+        min_value=0,
+        max_value=1,
+    )
+    return {"x_factor": 0, "y_factor": factor}
+
+
+def translate_x_policy(magnitude, magnitude_std):
+    # TODO(lukewood): should we integrate RandomTranslation with `factor`?
+    return {"width_factor": magnitude / 10, "height_factor": 0}
+
+
+def translate_y_policy(magnitude, magnitude_std):
+    # TODO(lukewood): should we integrate RandomTranslation with `factor`?
+    return {"width_factor": 0, "height_factor": magnitude / 10}
+
+
+def cutout_policy(magnitude, magnitude_std):
+    factor = core.NormalFactorSampler(
+        mean=0.75 * (magnitude / 10.0),
+        standard_deviation=0.75 * (magnitude_std / 10.0),
+        min_value=0,
+        max_value=1,
+    )
+    return {"width_factor": factor, "height_factor": factor}
+
+
+policy_pairs = [
```

Comment:
Capitalize this so that we don't mix this local variables.

Reply:
Done